custom-remote-user-store-management/agent/target/classes/connection.properties

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
Please remove the target directory from the PR add a new entry at https://github.com/wso2-incubator/samples-is/blob/master/.gitignore#L34 as follows.
```custom-remote-user-store-management/target/```

Reply:
Fixed in 51041691a23b06016ed6fa558504cc92b89c40b3 and 8a4b4fe793b231e0bd48d328c076d4cfd9494c5e